client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -29,6 +29,8 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
     const [selectedSymptoms, setSelectedSymptoms] = React.useState<string[]>([]);
     const [selectedBackgroundDiseases, setSelectedBackgroundDiseases] = React.useState<string[]>([]);
     const [otherSymptomChecked, setOtherSymptomChecked] = React.useState<boolean>(false);
+    const [otherBackgoundIllnessChecked, setOtherBackgoundIllnessChecked] = React.useState<boolean>(false);
+    const [otherBackgoundIllness, setOtherBackgoundIllness] = React.useState<string>('');
```

Comment:
Misspell - you spelled background wrong
maybe add 'is' before the 'otherBackgoundIllnessChecked' to mention that this state is boolean -(isOtherBackgoundIllnessChecked)